src/bin/travis/resources/local_centaur.conf

```diff
@@ -22,13 +22,19 @@ call-caching {
 system.graceful-server-shutdown = true
 
 backend.providers {
+  Local {
+    config {
+      script-epilogue = "sleep 5 && sync"
```

Comment:
😧

Reply:
I'm still messing around with this. Previous attempts to `sleep 1`, before or after `sync`, were still often failing to find files during `call_cache_capoeira`.